### PR TITLE
Feature#03/create in html and css the home view

### LINKED ISSUE #5: Create in html and css the home view
# REQUIREMENTS
Design the structure for the home view (Check In / Check Out)
- Make a numeric pad to introduce the secret number
- Make a simple clock to view the actual time

# DESIGN
image

### PR DESCRIPTION
**Pull request by Frcomparan**
# Description
What did you implemented/Why did you implemented this:
 - What?
   - We build the structure of the view to make the check (in/out) of employees (view for a tablet)
   - We added a numeric keyboard to introduce the private code
   - We added an option to see the current time
 - Why?
   - To have a base or template that allow employees to type their private codes
   - To have a template that would be to used save the check (in/out) of employees in the DB

 # Testing
 We only build the structure of the view (html and css), tests are not necessary

## Design or Mockup of the view
image
## Result of the build
image

### CODE REVIEW

File: app/views/employees/check.html.erb [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+      </div>
+
+      <button class="primary-button">Admin</button>
+
+    </div>
+    
+  </main>
+
+  
+
```
Comment (YilKanda) [inline app/views/employees/check.html.erb]: Remove unnecessary spaces in this file.

File: config/importmap.rb [hunk 2]
```diff
@@ -5,3 +5,4 @@
 pin "@hotwired/stimulus", to: "stimulus.min.js", preload: true
 pin "@hotwired/stimulus-loading", to: "stimulus-loading.js", preload: true
 pin_all_from "app/javascript/controllers", under: "controllers"
+# pin_all_from "app/javascript/custom", under: "custom"
```
Comment (YilKanda) [inline config/importmap.rb]: Code that you do not use, do not comment on it, delete it.

File: app/javascript/application.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
+// import "custom/main"
```
Comment (YilKanda) [inline app/javascript/application.js]: Code that you do not use, do not comment on it, delete it.

**Review by xFerchoVJ — APPROVED** [review]
LGTM

File: app/views/employees/check.html.erb (hunk 1 above)
Comment (Frcomparan) [reply app/views/employees/check.html.erb]: Thanks for the review, we will make the changes.

File: config/importmap.rb (hunk 2 above)
Comment (Frcomparan) [reply config/importmap.rb]: Perfect, we will eliminate the unnecessary code.

File: app/javascript/application.js (hunk 3 above)
Comment (Frcomparan) [reply app/javascript/application.js]: Perfect, we will eliminate the unnecessary code.

**Review by LuisGomez89 — APPROVED** [review]

**Review by alexs — APPROVED** [review]
LGTM 👏🏻👍🏻

**Review by MrCano369x — APPROVED** [review]
LGTM

**Pull request merged**